src/main/resources/application.yml

```diff
@@ -410,7 +410,7 @@ idam:
     microservice: probate_backend
 
 send-letter:
-  url: ${SEND_LETTER_SERIVCE_BASEURL:http://localhost:8485}
+  url: ${SEND_LETTER_SERIVCE_BASEURL:http://rpe-send-letter-service-aat.service.core-compute-aat.internal/}
```

Comment:
Are you sure this is okay? I thought we weren't using internal URLs anymore

Reply:
sorry - needs to be reverted anyway